tests/AbstractErrorHandlerTest.php

```diff
@@ -12,27 +12,23 @@
 namespace Sentry\Tests;
 
 use PHPUnit\Framework\TestCase;
-use Sentry\Client;
 
 abstract class AbstractErrorHandlerTest extends TestCase
 {
     /**
-     * @var \PHPUnit_Framework_MockObject_MockObject|Client
+     * @var \PHPUnit_Framework_MockObject_MockObject
      */
-    protected $client;
+    protected $callbackMock;
 
     protected function setUp()
     {
-        $this->client = $this->getMockBuilder(Client::class)
-            ->disableOriginalConstructor()
-            ->setMethodsExcept(['translateSeverity'])
-            ->getMock();
+        $this->callbackMock = $this->createPartialMock(\stdClass::class, ['__invoke']);
```

Comment:
As said elsewhere, you should use a data provider here to test every possible kind of callback